source/openapi3/split.yaml

```diff
@@ -1505,6 +1505,22 @@ tags:
       When Zepto is unable to credit funds to a recipient, we will automatically create a payout reversal credit back to the payer.
       Furthermore, within the payout reversal credit, Zepto will include details in the `description` and under the `reversal_details` key
       as to why the original credit to the recipient failed.
+  - name: Payouts
+    description: >
+      This endpoint gives you some control over a transaction:
+
+
+      * After it has been created; and
+
+      * Before it has been submitted to the banks; or
+
+      * If it was Prefailed, due to insufficient funds, and was therefore not submitted to the banks.
```

Comment:
@henripryde400 @chrisixion do we have prefail in NZ?

Reply:
Line removed as per standup discussion this morning